### ISSUE TITLE
Ele Custom Skin v3.1.5 Beta 1

### ISSUE THREAD

**Issue by dudaster** [dudaster/ele-custom-skin#709]
== Changelog ==

= 3.1.5 =
* Fixed issue with Custom Grid tab missing when the woocommerce is active.
* Fixed deprecated erros with the new Elementor version.

[ele-custom-skin-v3.1.5-beta1.zip](https://github.com/dudaster/ele-custom-skin/files/8223259/ele-custom-skin-v3.1.5-beta1.zip)

**Comment by pbaddock** [comment]
Quick feedback on this - the beta fixed the issue / error showing (when I enable WP debug) - seemingly a complete fix.    I noticed however that the original problem that lead me to find this issue / fix - was that at the elementor editor itself was not loading (checked memory sufficient etc), and that still appears to be occurring for me.   

Sequence of events was:
- editor not loading (I upped memory - still not loading)
- disabling plugins until i traced issue to ele custom skin
- enabled WPDEBUG - found deprecated error
- searched in github and found this issue here
- implemented beta 3.1.5 - debug deprecated error fixed / no longer showing

.....after that however, thinking the issue was fully resolved I turned off WPDEBUG but I noticed that the elementor editor no longer loads again on my site.

I turn WPDEBUG to true - and elementor editor loads (but no debug message/error shown).

So there is "some" indication that perhaps this issue is not fully resolved for me at least?
